convert2rhel/pkghandler.py

```diff
@@ -524,3 +526,19 @@ def install_additional_rhel_kernel_pkgs(additional_pkgs):
 def is_rhel_kernel_installed():
     installed_rhel_kernels = get_installed_pkgs_by_fingerprint(system_info.fingerprints_rhel, name="kernel")
     return len(installed_rhel_kernels) > 0
+
+
+def clear_yum_versionlock():
+    loggerinst = logging.getLogger(__name__)
+
+    if os.path.isfile(_VERSIONLOCK_FILE_PATH) and os.path.getsize(_VERSIONLOCK_FILE_PATH) > 0:
+        VERSIONLOCK_FILE.backup()
+
+        loggerinst.warn("yum versionlock plugin is in use. It may cause the conversion to fail.")
+        loggerinst.info("Do you want to continue? This will clear package version locks.")
```

Comment:
I know that this has been changed two times already but the question _Do you want to continue?_ is kind of duplicate because `ask_to_continue()` prints _Continue with the system conversion? [y/n]_.
I actually liked the previous _"If you choose to continue, the package version locks will be cleared."_. But since you don't want to use the passive form, @SpyTec, what about:
```suggestion
        loggerinst.info("Upon continuing, we will clear all package version locks.")
```
I'm not sure if "we" is the best choice, but it's hard to choose something here when not using the passive form.

Reply:
You're right, I thought it was just printing out `[y/n]` so we could be flexible. But yeah, that change SGTM. We should be using active voice as much as possible as it conveys better and sets a more direct tone